src/handlers/actions/makeArbitraryTransaction.ts

```diff
@@ -52,6 +56,7 @@ export default async (event: ContractEvent): Promise<void> => {
   await writeActionFromEvent(event, colonyAddress, {
     type: ColonyActionType.MakeArbitraryTransaction,
     initiatorAddress: receipt.from,
+    recipientAddress: receipt.to,
```

Comment:
Looks like this is adding the field that was removed in #304 - is this intentional?